docs/install-pro.md

```diff
@@ -0,0 +1,115 @@
+# Install Percona XtraDB Cluster Pro
+
+--8<--- "pro-build-announcement.md"
```

Comment:
You can remove this one. 
I like the snippet closer to the instructions. Better chance that they will read it.

Reply:
removed